<launch>
    <node pkg="rm_fc" type="vins_to_mavros" name="vins_to_mavros" output="screen">
        <remap from="~vision_pose" to="/mavros/vision_pose/pose" />
        <remap from="~odom" to="/vins_node/odometry" />
    </node>
</launch>